paimon-web-server/src/main/docker/Dockerfile

```diff
@@ -0,0 +1,30 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+FROM eclipse-temurin:8-jre
```

Comment:
Why use this image?

Reply:
eclipse-temurin:8-jre is a Docker image based on the Eclipse Temurin project, specifically providing the Java 8 Runtime Environment (JRE). Eclipse Temurin is part of the Adoptium project, which carries on the work from AdoptOpenJDK, delivering open-source, rigorously tested, and OpenJDK-compatible binary distributions. When you see eclipse-temurin:8-jre, it refers to a lightweight runtime environment tailored for Java 8, intended for scenarios where the full development kit (JDK) is not necessary. It includes only the components required to run Java applications, such as the Java Virtual Machine (JVM) and core class libraries.

Reply:
```
ARG java_image_tag=8-jdk-focal
FROM eclipse-temurin:${java_image_tag}
```